### PR TITLE
Feature/control player page

### PR DESCRIPTION
**Pull request by romain130492**
Please describe the purpose of the pull request.

* [notion page](https://www.notion.so/akkadu/RSI-API-pause-play-programmatically-the-VM-player-49e7785ec3834236a09d4f5ad1803f3f)
* For the interpretation player we want to be able to give the option to play/pause programmatically the player of the virtual platforms.

## How To Test

* Go on this page : https://deploy-preview-2--rsi-akkadu-vue-demo.netlify.app/#/and start the player,
* When selecting the `interpreted language` (here chinese), it should `mute` the video
* when selecting the `floor language` (english) it shoud `unmute` the video.

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have removed all unused css, language strings, variables, functions, images and other resources.
- [ ] My change requires to update dependencies and I have updated `package.json` and `yarn.lock` accordingly.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

加油！

### CODE REVIEW

File: modules/rsi-api-interpretation-player/src/index.ts [hunk 1]
```diff
@@ -33,10 +37,48 @@ export default class InterpretationPlayer extends RSIBase {
     }
   }
   async init() {
-    const gatewayReponse = /* await */ this.gatewayRequest(this.apiKey, this.roomName);
+    this.gatewayResponse = /* await */ this.gatewayRequest(this.apiKey, this.roomName);
```
Comment (deevannaveed) [inline modules/rsi-api-interpretation-player/src/index.ts:40]: if we are not using `await`, we may remove the comment and also remove the async keyword.

File: modules/rsi-api-interpretation-player/src/index.ts [hunk 2]
```diff
@@ -33,10 +37,48 @@ export default class InterpretationPlayer extends RSIBase {
...
+       video.muted = isMuted
+    });
+  }
+  /**
+   * @description Get the video player of the Virtual Platform page
+   * @private
+   */
+  getVideoPlayerVP(){
+    if(!document){
+      console.warn('getVideoPlaterVP(), document is not defined');
```
Comment (deevannaveed) [inline modules/rsi-api-interpretation-player/src/index.ts:77]: There is a typo in the warning message `getVideoPlaterVP` should be `getVideoPlayerVP`.

File: modules/rsi-api-interpretation-player/src/index.ts [hunk 3]
```diff
@@ -33,10 +37,48 @@ export default class InterpretationPlayer extends RSIBase {
...
+      return
+    }
+    return document.getElementsByTagName('video');
```
Comment (deevannaveed) [inline modules/rsi-api-interpretation-player/src/index.ts:80]: Why are we getting all the `video` tags from the document/page, can we use some class for our injected videos?

**Review by deevannaveed — CHANGES_REQUESTED** [review]
Nice work @romain130492, I have the following comments. 

1. I cannot see the player on the page, see the screenshot below. 
image

2. What is the purpose of the `modules/rsi-api-react/example/build/*` files, do we need them or can be git ignored?

File: modules/rsi-api-interpretation-player/src/index.ts (hunk 3 above)
Comment (romain130492) [reply modules/rsi-api-interpretation-player/src/index.ts:80]: thats not our video actually, the purpose is to get the video added by the Virtual platform on their pages. Then we want to mute/unmute **their** video

File: modules/rsi-api-interpretation-player/src/index.ts (hunk 2 above)
Comment (romain130492) [reply modules/rsi-api-interpretation-player/src/index.ts:77]: thanks updating that one

File: modules/rsi-api-interpretation-player/src/index.ts (hunk 1 above)
Comment (romain130492) [reply modules/rsi-api-interpretation-player/src/index.ts:40]: we will use it in the future once the gateway method is set up

Comment (romain130492) [reply modules/rsi-api-interpretation-player/src/index.ts:40]: yes im going to add them to the gitignore

**Comment by romain130492** [comment]
You have to use the preview link I added in the description, otherwise you are seeing the main branch : https://deploy-preview-2--rsi-akkadu-vue-demo.netlify.app/#/and

**Review by deevannaveed — APPROVED** [review]
Thanks @romain130492  looks good.

**Pull request merged**